dds_web/security/project_user_keys.py

```diff
@@ -151,36 +185,37 @@ def __owner_identifier(owner):
     return owner.email if isinstance(owner, models.Invite) else owner.username
 
 
-def __encrypt_owner_private_key(owner, private_key, temporary_key=None):
-    if temporary_key is None:
-        temporary_key = ciphers.aead.AESGCM.generate_key(bit_length=256)
+def __encrypt_owner_private_key(owner, private_key, owner_key=None):
+    key = owner_key
+    if key is None:
+        key = ciphers.aead.AESGCM.generate_key(bit_length=256)
```

Comment:
`key = owner_key or ciphers.aead.AESGCM.generate_key(bit_length=256)`

Reply:
I find this an unsafe way of writing and wouldn't recommend it, because it is not a boolean variable, however it's your choice. changed!